agent/scripts/generate-ui-cache.js

```
@@ -1,16 +1,35 @@
 const fs = require('fs');
```

ESLint

[error] 'require' is not defined.

(no-undef)